webapp/backend/src/handlers/index.ts

```diff
@@ -0,0 +1,15 @@
+import {Request, Response, Router} from "express";
+import demoVideos from "./api/videos";
+
+// Router object
+const routes = Router();
+
+// Main route
+routes.get("/", (req: Request, res: Response) => {
+    res.send("Welcome to NSMQ AI Web backend")
+});
+
+// Demo video route
+routes.use("/videos", demoVideos);
```

Comment:
we will also have live video endpoints so let's use `demo-videos` or more applicable endpoint name to differentiate them?

Reply:
> we will also have live video endpoints so let's use `demo-videos` or more applicable endpoint name to differentiate them?

Sure. Thanks for the feedback.